src/WebJobs.Script/Config/ScriptApplicationHostOptions.cs

```diff
@@ -20,6 +20,8 @@ public class ScriptApplicationHostOptions
 
         public string TestDataPath { get; set; }
 
+        public bool InStandbyMode { get; set; } = false;
```

Comment:
I don't think we want this to be part of our public options surface area. Can you make this internal?

Reply:
I agree that this isn't ideal. This afternoon I tried to break InStandbyMode into its own Options object and make this setup use that instead. But it's introducing some interesting timing issues with the Options cache invalidation (it's happening in the wrong order). Going to think through it again and see if there's another way to break this out...